### PR TITLE
Expanding variable_utils

### PR DESCRIPTION
**Pull request by loumalouomega**
Adding Non-Historical variables utilities. Clean up

### CODE REVIEW

File: kratos/utilities/variable_utils.h [hunk 1]
```diff
@@ -183,16 +189,63 @@ class VariableUtils
...
+        KRATOS_CATCH("")
+    }
+    
+    /**
+     * Takes the value of a non-historical vector variable and sets it in other non-historical variable
+     * @param OriginVariable: reference to the origin vector variable
+     * @param SavedVariable: reference to the destination vector variable
+     * @param rNodes: reference to the objective node set
+     */
+    void SaveVectorNonHistVar(
```
Comment (rubenzorrilla) [inline kratos/utilities/variable_utils.h]: I'd have that the variable is nodal in the method name (something like SaveVectorNonHistNodevar). However, this naming convention matches the non historical variable method above. Maybe we can discuss about renaming the existent ones.

File: kratos/utilities/variable_utils.h [hunk 2]
```diff
@@ -183,16 +189,63 @@ class VariableUtils
...
+        KRATOS_CATCH("")
+    }
+
+    /**
+     * Takes the value of a non-historical scalar variable and sets it in other non-historical variable
+     * @param OriginVariable: reference to the origin scalar variable
+     * @param SavedVariable: reference to the destination scalar variable
+     * @param rNodes: reference to the objective node set
+     */
+    void SaveScalarNonHistVar(
```
Comment (rubenzorrilla) [inline kratos/utilities/variable_utils.h]: Same naming comment.

**Review by rubenzorrilla — CHANGES_REQUESTED** [review]
Since you have formatted all the file, take the chance to put the { inline as the convention states.

**Comment by loumalouomega** [comment]
Ping

**Comment by RiccardoRossi** [comment]
is this urgent? because the "variable_utils" is being modified in  #943 and you are going to create a huge conflict...

**Review by RiccardoRossi — CHANGES_REQUESTED** [review]
also i don't know what you did but it is impossible to understand what changes you did in variable_utils.h

**Comment by loumalouomega** [comment]
@rubenzorrilla Ask me to set the keys, so I used the clang-format

**Pull request closed without merging**